portal-frontend/src/app/features/notice-of-intents/edit-submission/land-use/land-use.component.html

```diff
@@ -0,0 +1,307 @@
+<section>
+  <div class="step-description">
+    <h2>Land Use</h2>
+    <p>Current land use of parcel(s) under notice of intent and adjacent parcels.</p>
+    <p>*All fields are required unless stated optional.</p>
+  </div>
+  <app-warning-banner>
+    Please consult the
+    <a
+      href="https://www.alc.gov.bc.ca/application-and-notice-process/applications/what-the-commission-considers/"
```

Comment:
is there custom link for noi or this one includes both?

Reply:
Confirmed with Ansley this is correct